multisrc/overrides/madara/randomscan/src/luratoon.kt

```diff
@@ -7,9 +7,9 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class RandomScan : Madara(
-    "Random Scan",
-    "https://randomscanlators.net",
+class LuraToon : Madara(
```

Comment:
Match the class' name with the filename, preserving casing. The filename should have been `LunaToon.kt`.

Reply:
why "LunaToon.kt" if the site has changed to LuraToon?

Reply:
> why "LunaToon.kt" if the site has changed to LuraToon?

That was a typo. The point is that for single-class files, the filename should be `{class name}.kt`. In this case, `LuraToon.kt`.

Reply:
okay then, I'll change the file name.